mcstatus/bedrock_status.py

```diff
@@ -11,7 +11,10 @@
 
 
 class BedrockServerStatus:
-    request_status_data = b"\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff\xff\x00\xfe\xfe\xfe\xfe\xfd\xfd\xfd\xfd\x124Vx"
+    request_status_data = bytes.fromhex(
+        # see https://wiki.vg/Raknet_Protocol#Unconnected_Ping
+        "01" + "000000000000000000" + "ffff00fefefefefdfdfdfd12345678" + "a20871356d207e61"  # fmt: skip
```

Comment:
```suggestion
        "01" + "000000000000000000" + "ffff00fefefefefdfdfdfd12345678" + "a20871356d207e61"  # fmt: skip
```

The PR looks ok, however we could replace the random data with all 0es instead. I know it's supposed to be something random, but all zeroes are one of the things full randomized data can give you. 

Though hard-coding some gibberish like this doesn't help make it clear what this is, and it actually makes it appear as if this data was somehow deliberately chosen, when it's really just random. Let's use 0es for now.

Reply:
yes, it is quite possible to replace, this method is used and works